spec/sql/billing/bill_cdr_batch_spec.rb

```diff
@@ -88,30 +70,58 @@
         term_gw_id: gw.id,
         duration: -11 # should be skipped in stats
       }
-    ].to_json.to_s
```

Comment:
`batch_data` being conerted to json in the `subject`.  
FYI no need to call `to_json.to_s`, the `to_json` method already return a string